src/KeyVault/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs256.cs

```diff
@@ -20,38 +25,50 @@ public Rs256()
         {
         }
 
-        public override byte[] SignHash( AsymmetricAlgorithm key, byte[] digest )
+        public override ISignatureTransform CreateSignatureTransform( AsymmetricAlgorithm key )
         {
-            if ( key == null )
-                throw new ArgumentNullException( "key" );
+            return new Rs256SignatureTransform( key );
+        }
```

Comment:
Sorry, incomplete reply; this is a 2.n package change so it is already a major version bump; the 2.n versions of the AKV Core, Crypto and Extensions packages have not been shipped yet.